src/tiffimage_int.cpp

```diff
@@ -1529,21 +1684,45 @@ namespace Exiv2 {
         {    0x0005, canonId,          EXV_SIMPLE_BINARY_ARRAY(canonPaCfg)       },
         {    0x000f, canonId,          EXV_SIMPLE_BINARY_ARRAY(canonCfCfg)       },
         {    0x0012, canonId,          EXV_SIMPLE_BINARY_ARRAY(canonPiCfg)       },
+    //  {    0x0026, canonId,          EXV_SIMPLE_BINARY_ARRAY(canonAf2Cfg)      }, // allow TiffDecoder::decodeCanonAFInfo to deal with this
         {    0x0035, canonId,          EXV_SIMPLE_BINARY_ARRAY(canonTiCfg)       },
         {    0x0093, canonId,          EXV_BINARY_ARRAY(canonFiCfg, canonFiDef)  },
-        {    0x00a0, canonId,          EXV_SIMPLE_BINARY_ARRAY(canonPrCfg)  },
+        {    0x00a0, canonId,          EXV_SIMPLE_BINARY_ARRAY(canonPrCfg)       },        
+        {    0x4013, canonId,          EXV_SIMPLE_BINARY_ARRAY(canonAfMiAdjCfg)  },
+    //  {    0x4015, canonId,          EXV_SIMPLE_BINARY_ARRAY(canonVigCorCfg)   },
```

Comment:
Why is this one commented out? I think we should add a comment explaining the reason, so we don't forget.

Reply:
I don't know.  @alexvanderberkel did that.  My change to that line was cosmetic (aligning everything).